src/main/kotlin/calculator/Parser.kt

```diff
@@ -0,0 +1,30 @@
+package calculator
+
+object Parser {
+    private val defaultDelimiters: List<String> = listOf(",", ":")
+    private var delimiters: List<String> = defaultDelimiters
+
+    private fun addDelimiters(vararg delimiters: String) {
+        this.delimiters = this.delimiters.plus(delimiters)
+    }
+
+    private fun extractCustomDelimitersIfExist(value: String): String {
```

Comment:
```suggestion
    private fun extractNumbers(value: String): String {
```

커스텀한지, 존재하는지와 같은 세부 구현사항을 함수 네이밍에 모두 나타낼 필요 없어요.  
결국 이 함수는 문자열에서 `피연산자(숫자들)`를 도출해내는 기능을 합니다.  

세부 구현사항보다는 `결과값`을 함수 이름에 녹여내는 연습을 해볼까요?😉

Reply:
`함수는 1가지 일만 해야한다` 원칙을 지켜볼까요?  

현재 해당 함수는 2가지 일을 하고 있습니다.  

1) delimeter 뽑아내서 추가
2) 숫자 뽑아내서 반환

Reply:
세부 구현사항은 변하기 쉽기에 더더욱 세부 구현사항을 함수 네이밍에 나타내는 것을 지향해야 할 것 같네요!
또한 `함수는 1가지 일만 해야한다 원칙`도 반영하도록 수정해보았습니다! 17b97837eb93ec4462b9ac34663a5e3b25995f66

좋은 의견 감사합니다 !!